compat/media/Android.mk

```diff
@@ -122,6 +122,13 @@ LOCAL_SHARED_LIBRARIES := \
 	libaudioutils \
 	libmediaplayerservice
 
+ifeq ($(IS_ANDROID_8),true)
```

Comment:
Where is that IS_ANDROID_8 defined, I only see IS_ANDROID_5 above in the same file. Should this be added?

Reply:
Good catch, I rebased our whole Halium tree on top of master now and got it, will push a separate PR and close this one.